test/test_baza-rb.rb

```diff
@@ -748,6 +749,288 @@ def test_durable_load_from_sinatra
     end
   end
 
+  def test_download_updates_host_from_x_zerocracy_host_header
+    WebMock.disable_net_connect!
+    Dir.mktmpdir do |dir|
+      file = File.join(dir, 'test.txt')
+      initial_host = 'example.org'
```

Comment:
@paveltrty see https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html